bigbluebutton-html5/imports/ui/components/app/component.jsx

```diff
@@ -124,8 +130,9 @@ class App extends Component {
 
     return (
       <Resizable
-        minWidth="10%"
-        maxWidth="20%"
+        defaultSize= {{ width: (window.innerWidth * (USERLIST_DEFAULT_WIDTH_RELATIVE / 100.0)) < USERLIST_MAX_WIDTH_PX ? USERLIST_DEFAULT_WIDTH_RELATIVE + "%" : USERLIST_MAX_WIDTH_PX }}
```

Comment:
Please move the defaultSize calculation to a variable outside of the  <Resizable> tag

Reply:
Ok